### ISSUE TITLE
Improvement: Stripe Connect email receipts

### ISSUE THREAD

**Issue by gpalmas** [ITPrism/CrowdFunding#209]
Hi Todor. 
As I can see here https://support.stripe.com/questions/email-receipts Stripe can send an email receipt to donors in case of successiful payment or refund.
But, as you can see in attached images, the Stripe Connect Plugin is not passing (to Stripe) the email of the donors in the right way.
The email is loaded by Stripe as customer name and not as customer email address. Then, Stripe can't send his own receipt to donors because the email field is empty.
I don't know if it's a choice or a mistake.
I think that it might be useful to increase the donors "level of trust" about the website...

Best regards!

**Comment by ITPrism** [comment]
Hi Giuseppe,
It is really good suggestion. I like it.
Thank you very much! 
I am going to do a research and to implement the feature if it is possible.

**Comment by ssnobben** [comment]
Yep a good suggestion and Stripe is a very good payment processor to support..

**Comment by ssnobben** [comment]
Todor is there any eta of when 2.7.0 will be out?  https://github.com/ITPrism/CrowdFunding/blob/master/CHANGELOG.md
